daprdocs/content/en/operations/support/support-versioning.md

```diff
@@ -0,0 +1,102 @@
+---
+type: docs
+title: "Versioning policy"
+linkTitle: "Versioning "
+weight: 200
+description: "Dapr's versioning policies"
+---
+
+## Introduction
+Dapr is designed for future changes in the runtime, APIs and components with versioning schemes. This topic describes the versioning schemes and strategies for APIs, manifests such as components and Github repositories. 
+
+## Versioning
+Versioning is the process of assigning either unique version names or unique version numbers to unique states of computer software. 
+- Versioning provides compatibility, explicit change control and handling changes, in particular breaking changes.
+- Dapr strives to be backwards compatible. If a breaking change is needed it’ll be [announced in advance]({{< ref "support-release-policy#feature-and-deprecations" >}}).
+- Deprecated features are done over multiple releases with both new and deprecated features working side-by-side.
+
+
+Versioning refers to the following Dapr repos: dapr, CLI, stable language SDKs, dashboard, components-contrib, quickstarts, helm-charts and documentation.
+
+Dapr has the following versioning schemes:
+- Dapr `HTTP API` versioned with `MAJOR.MINOR` 
+- Dapr `GRPC API` with `MAJOR`
+- Releases (GitHub repositories including dapr, CLI, SDKs and Helm Chart) with `MAJOR.MINOR.PATCH`
+- Documentation and Quickstarts repositories are versioned with the Dapr runtime repository versioning.
+- Dapr `Components` with `MAJOR` in component-config GitHub repositories.
+- Dapr `Manifests` with `MAJOR.MINOR`. These include subscriptions and configurations.
+
+Note that the Dapr APIs,  binaries releases (runtime, CLI, SDKs) and components are all independent from one another.
+
+## Dapr API 
+The Dapr HTTP API is versioned according to these [REST API guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#71-url-structure). 
+
+Based to the these guidelines; 
+- A `MAJOR` version of the API is incremented when a deprecation is expected of the older version. Any such deprecation will be communicated and an upgrade path made available.
+- A `MINOR` versions *may* be incremented for any other changes. For example a change to the JSON schema of the message sent to the API.
+The definition of a breaking change to the API can be viewed [here](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#123-definition-of-a-breaking-change).
+- Experimental APIs include an “alpha” suffix to denote for their alpha status. For example v1.0alpha, v2.0alpha, etc.
+
+## Dapr runtime 
+Dapr releases use `MAJOR.MINOR.PATCH` versioning. For example 1.0.0. Read [Supported releases]({{< ref support-release-policy.md >}}) for more on the versioning of releases. 
+
+## Helm Charts
+Helm charts are versioned with the Dapr runtime.
```

Comment:
We should link to either the helm charts repo, or other docs on helm

Reply:
Done